### PR TITLE
os_test: add case for setjmp/longjmp

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Depends on https://github.com/OP-TEE/optee_os/pull/753

### CODE REVIEW

**Comment by pascal-brand-st-dev** [comment]
`Reviewed-by: Pascal Brand <pascal.brand@linaro.org>`

File: ta/os_test/os_test.c [hunk 1]
```diff
@@ -619,6 +620,26 @@ static TEE_Result test_float(void)
 }
 #endif /*CFG_TA_FLOAT_SUPPORT*/
 
+static __noinline void call_longjmp(jmp_buf env)
+{
+	DMSG("Calling longjmp");
+	longjmp(env, 1);
+	EMSG("error: Longjmp returned to calling function");
```
Comment (jforissier) [inline ta/os_test/os_test.c]: Nit: `longjmp` not `Longjmp`

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: ta/os_test/os_test.c (hunk 1 above)
Comment (jenswi-linaro) [inline ta/os_test/os_test.c]: I'll  fix

**Comment by jenswi-linaro** [comment]
Tags applied.

**Pull request merged**